### DISCUSSION TITLE
BFE有计划进行go版本升级么？

### DISCUSSION THREAD

**Discussion by Lambor-Chai** [bfenetworks/bfe#1165] [Ideas（新功能的建议和讨论）]
go现在的主版本 已经升级到1.23版本；go 1.21 1.22版本引入了新的PGO和内存使用优化，BFE的主版本有计划升级么？

**Comment by cherishman2005** [comment]
可以分析bfe的http transport代码，期待版本升级。

https://github.com/bfenetworks/bfe/issues/1151

**Comment by Lambor-Chai** [comment]
您的来信已收到，我会尽快查看并回复，祝您生活愉快！
